src/components/footer/www/footer.jsx

```diff
@@ -4,165 +4,206 @@ var FormattedMessage = require('react-intl').FormattedMessage;
-                                <FormattedMessage id='general.scratchConference' />
-                            </a>
-                        </dd>
-                        <dd>
-                            <a href="http://www.scratchfoundation.org/">
-                                <FormattedMessage id='general.scratchFoundation' />
-                            </a>
-                        </dd>
-                    </dl>
-                </div>
+                        <dl>
+                            <dt>
+                                <FormattedMessage id='general.legal'/>
+                            </dt>
+                            <dd>
+                                <a href="/terms_of_use/">
+                                    <FormattedMessage id='general.termsOfUse' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="/privacy_policy/">
+                                    <FormattedMessage id='general.privacyPolicy' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="/DMCA/">
+                                    <FormattedMessage id='general.dmca' />
+                                </a>
+                            </dd>
+                        </dl>
 
+                        <dl>
+                            <dt>
+                                <FormattedMessage id='footer.scratchFamily' />
+                            </dt>
+                            <dd>
+                                <a href="http://scratched.gse.harvard.edu/">
+                                    <FormattedMessage id='general.scratchEd' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="http://www.scratchjr.org/">
+                                    <FormattedMessage id='general.scratchJr' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="http://day.scratch.mit.edu/">
+                                    <FormattedMessage id='general.scratchday' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="/conference/">
+                                    <FormattedMessage id='general.scratchConference' />
+                                </a>
+                            </dd>
+                            <dd>
+                                <a href="http://www.scratchfoundation.org/">
+                                    <FormattedMessage id='general.scratchFoundation' />
+                                </a>
+                            </dd>
+                        </dl>
+                    </div>
+                </MediaQuery>
```

Comment:
I'm not convinced that this will be an example the rest of the site has to follow. This feels more like a unique case than a standard case, given the way the content reduces.

The only other way this could feasibly work, from what i can see, is if we were to restructure the footer so that the column headers were their own `FlexRow`, and then all the rest of the content was in one giant `FlexRow` that was column oriented, and overflowed into the next column. Then you could potentially use `display: none`.

However, that might be really tricky to line up correctly with column headers.